Makefile

```diff
@@ -82,9 +82,25 @@ github_actions:
 build-system-deps:
 	$(PYTHON) -m pip install wheel
 
+# TODO: change DOC to DOC301 - ORQSDK-965
```

Comment:
(see my suggestion below about the rules we want to enforce)

Reply:
[64503bc](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/308/commits/64503bc2f4bba4d920a3cda53a2ad8b289c246d3)